[Bug Report] Playing video from youtube doesnt work

### Bug Description

When playing a youtube video
the bot doesnt play the audio and i get this traceback:
````
Jun 23 19:47:33 pi64-1 env[1297069]: [19:47:33] [ERROR] [LocalAudioTrackExecutor]: Error in playback of EpcDH9faOQw
Jun 23 19:47:33 pi64-1 env[1297069]: com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:44)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:116)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:348)
Jun 23 19:47:33 pi64-1 env[1297069]:         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
Jun 23 19:47:33 pi64-1 env[1297069]:         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
Jun 23 19:47:33 pi64-1 env[1297069]:         at java.base/java.lang.Thread.run(Thread.java:829)
Jun 23 19:47:33 pi64-1 env[1297069]:         Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation:
Jun 23 19:47:33 pi64-1 env[1297069]:   lavaplayer.version: 1.3.78
Jun 23 19:47:33 pi64-1 env[1297069]:   os.arch: aarch64
Jun 23 19:47:33 pi64-1 env[1297069]:   os.name: Linux
Jun 23 19:47:33 pi64-1 env[1297069]:   os.version: 6.1.21-v8+
Jun 23 19:47:33 pi64-1 env[1297069]:   java.vendor: Debian
Jun 23 19:47:33 pi64-1 env[1297069]:   java.version: 11.0.23
Jun 23 19:47:33 pi64-1 env[1297069]:   java.runtime.version: 11.0.23+9-post-Debian-1deb11u1
Jun 23 19:47:33 pi64-1 env[1297069]:   java.vm.version: 11.0.23+9-post-Debian-1deb11u1
Jun 23 19:47:33 pi64-1 env[1297069]: Caused by: java.lang.RuntimeException: Not success status code: 403
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:87)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:116)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:105)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.internalRead(PersistentHttpStream.java:150)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:166)
Jun 23 19:47:33 pi64-1 env[1297069]:         at java.base/java.io.DataInputStream.readByte(DataInputStream.java:270)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaEbmlReader.readEbmlInteger(MatroskaEbmlReader.java:39)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaFileReader.readNextElement(MatroskaFileReader.java:45)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.readFile(MatroskaStreamingFile.java:78)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.loadMatroskaFile(MatroskaAudioTrack.java:51)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:34)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:57)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:49)
Jun 23 19:47:33 pi64-1 env[1297069]:         at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:104)
Jun 23 19:47:33 pi64-1 env[1297069]:         ... 4 common frames omitted
````

### Steps to Reproduce

1. Start bot
2. Play any youtube video

### Expected Result

Audio to play / no traceback

### Debug Output

openjdk 11.0.23 2024-04-16
OpenJDK Runtime Environment (build 11.0.23+9-post-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 11.0.23+9-post-Debian-1deb11u1, mixed mode)

JMusicBot-0.4.1.jar

       _,met$$$$$gg.                                                                                                                                                                   g
    ,g$$$$$$$$$$$$$$$P.       ----------
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 11 (bullseye) aarch64
 ,$$P'              `$$$.     Host: Raspberry Pi 400 Rev 1.1
',$$P       ,ggs.     `$$b:   Kernel: 6.1.21-v8+
`d$$'     ,$P"'   .    $$$    Uptime: 23 days, 3 hours, 40 mins
 $$P      d$'     ,    $$P    Packages: 1531 (dpkg)
 $$:      $$.   -    ,d$$'    Shell: bash 5.1.4
 $$;      Y$b._   _,d$P'      Terminal: /dev/pts/0
 Y$$.    `.`"Y$$$$P"'         CPU: BCM2835 (4) @ 1.800GHz
 `$$b      "-.__              Memory: 766MiB / 3794MiB
  `Y$$
   `Y$$.
     `$$b.
       `Y$$b.
          `"Y$b._
              `"""



### Additional Info

_No response_

### Checklist

- [X] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [X] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

--- a/pom.xml
+++ b/pom.xml
@@ -31,9 +31,9 @@
             <url>https://m2.duncte123.dev/releases</url>
         </repository>
         <repository>
-            <id>arbjergDev-snapshots</id>
+            <id>arbjergDev</id>
             <name>Lavalink Repository</name>
-            <url>https://maven.lavalink.dev/snapshots</url>
+            <url>https://maven.lavalink.dev/releases</url>
         </repository>
     </repositories>
 
@@ -55,12 +55,12 @@
         <dependency>
             <groupId>dev.arbjerg</groupId>
             <artifactId>lavaplayer</artifactId>
-            <version>2.1.1</version>
+            <version>2.2.1</version>
         </dependency>
         <dependency>
-            <groupId>com.github.lavalink-devs</groupId>
-            <artifactId>lavaplayer-youtube-source</artifactId>
-            <version>1.0.4</version>
+            <groupId>dev.lavalink.youtube</groupId>
+            <artifactId>common</artifactId>
+            <version>1.4.0</version>
         </dependency>
 	<dependency>
 	    <groupId>com.github.jagrosh</groupId>

